### DISCUSSION TITLE
Changing the mouse sensitivity

### DISCUSSION THREAD

**Discussion by Shawarma22** [end-4/dots-hyprland#834] [Q&A] [answered]
Exactly as the title states, i want to change my mouse sens or specifically lower it, and also disable mouse acceleration as i feel thats turned on. I dont have any clue why the gnome settings get overridden as i changed my mouse sens there.

**Comment by end-4** [comment, marked as answer]
Changing gnome settings in most cases won't change things in hyprland. 
### Mouse accel
To always disable mouse accel, you should change hyprland config. Have `input:accel_profile` set to `flat`
if you want to disable mouse accel temporarily for your game, open the launcher and type `>raw`
###  Sensitivity
Change `input:sensitivity` in hyprland config

**Reply by Shawarma22** [reply]
Thank you, and sorry for the late reply. 👍
